### PR TITLE
[release-0.34] mtu bug

### PR DESCRIPTION
**Pull request by qinqon**
**What this PR does / why we need it**:

This PR cherry-picks https://github.com/kubevirt/kubevirt/pull/4289 to fix the MTU bug at masquerade binding.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
The helpers `tests/libnet/ipaddress.go` were introduced at a different PR that is not being cherry-picked, and the PR that fixes 
the mtu bug depends on that, so we copy those helpers manually at different commit at this PR.

**Release note**:

```release-note
Masquerade binding - set the virt-launcher pod interface MTU on the bridge.
```

### CODE REVIEW

File: tests/libnet/ipaddress.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package libnet
+
+import (
+	k8sv1 "k8s.io/api/core/v1"
+	netutils "k8s.io/utils/net"
+
+	v1 "kubevirt.io/client-go/api/v1"
+)
+
+func GetPodIpByFamily(pod *k8sv1.Pod, family k8sv1.IPFamily) string {
```
Comment (AlonaKaplan) [inline tests/libnet/ipaddress.go]: This method is not in use by the PR so we may remove it.

File: tests/vmi_networking_test.go [hunk 2]
```diff
@@ -831,9 +831,6 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 			})
 
 			table.DescribeTable("should have the correct MTU", func(ipFamily k8sv1.IPFamily) {
-				if ipFamily == k8sv1.IPv6Protocol {
-					libnet.SkipWhenNotDualStackCluster(virtClient)
```
Comment (AlonaKaplan) [inline tests/vmi_networking_test.go]: Why do you remove it? How would it work on single stack clusters?

**Comment by qinqon** [comment]
Removed `GetPodIpByFamily` and added `SkipWhenNotDualStackCluster`

File: tests/libnet/ipaddress.go (hunk 1 above)
Comment (qinqon) [reply tests/libnet/ipaddress.go]: Done

File: tests/vmi_networking_test.go (hunk 2 above)
Comment (qinqon) [reply tests/vmi_networking_test.go]: Done, the needed coded has been backported too.

File: BUILD.bazel [hunk 3]
```diff
@@ -36,6 +36,7 @@ goimports(
     exclude_paths = [
         "./vendor/*",
         "./.history/*",
+        "./.git/*",
```
Comment (AlonaKaplan) [inline BUILD.bazel:39]: Why this change is needed?

Comment (qinqon) [reply BUILD.bazel:39]: It's not suppose to be here, will remove it.

The story about this is here, https://github.com/kubevirt/kubevirt/pull/4330

**Comment by qinqon** [comment]
Removed .git/* bazel's goimport

**Review by EdDev — APPROVED** [review]
